.github/workflows/Release.yml

```diff
@@ -26,9 +26,15 @@ jobs:
           sed --in-place -e "s/-SNAPSHOT/-$(date +%Y%m%d.%H%M)-$(git rev-parse --short HEAD)/" gradle.properties
           cat gradle.properties
 
+      - name: Configure JDK
```

Comment:
copied from tempest, do we want this?

Reply:
This should track whatever the Build.yml does